dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -209,6 +209,11 @@ public void addTasksTo(List<? super Task<?>> out, @Nonnull ConnectorArguments ar
         "Extracting query logs from tables: '{}' and '{}'",
         tableNames.queryLogsTableName(),
         tableNames.sqlLogsTableName());
+
+    out.add(
+        new TeradataTablesValidatorTask(
+            tableNames.queryLogsTableName(), tableNames.sqlLogsTableName()));
```

Comment:
What about other tables? There are 5 important tables that can be overriden for TD:
  -Dteradata-logs.query-logs-table=pdcrdata.QryLogV_hst
  -Dteradata-logs.sql-logs-table=pdcrdata.DBQLSqlTbl_hst
  -Dteradata-logs.utility-logs-table=pdcrdata.DBQLUtilityTbl_hst
  -Dteradata-logs.res-usage-scpu-table=pdcrdata.ResUsageScpu_hst
  -Dteradata-logs.res-usage-spma-table=pdcrdata.ResUsageSpma_hst

Reply:
I suggest to do it in a separate PR. I already created a ticket for that.